src/components/Calculator.js

```diff
@@ -1,41 +1,63 @@
-/* eslint-disable arrow-body-style */
-/* eslint-disable react/prefer-stateless-function */
+/* eslint-disable react/prop-types */
 import React from 'react';
+import calculate from '../logic/calculate';
```

Comment:
Good job so far on your project but there are a few changes the previous reviewer did not aware of.

- [ ] Kindly remove the ESLint disabling on prop-types. The ESLint check helps you write error-free code. Disabling it totally defies its purpose. Check this article on [Prop Types](https://www.freecodecamp.org/news/how-to-use-proptypes-in-react/). Please fix the prop types error that ESLint is giving and resubmit for a review 👍

Reply:
@AmaduKamara  yeah dear reviewer that is true, it is ready